coral/src/app/features/connectors/details/documentation/components/NoDocumentationBanner.tsx

```diff
@@ -0,0 +1,25 @@
+import { EmptyState, EmptyStateLayout } from "@aivenio/aquarium";
```

Comment:
We probably could move that into shared components since we're reusing it, when we add a prop for the text. Not needed for this PR, but may be a nice first issue?

Reply:
Yes sounds good ! Will create it after the PR is merged so I can point to the two files in `main`